kairos-cli/bin/crypto/private_key.rs

```diff
@@ -0,0 +1,15 @@
+use crate::crypto::error::CryptoError;
+
+pub struct CasperPrivateKey(pub casper_types::SecretKey);
+
+impl CasperPrivateKey {
+    pub fn from_key(public_key: casper_types::SecretKey) -> Self {
```

Comment:
this method is shallow, i.e. what is the difference between calling
```
let casper_key = CasperPrivateKey(public_key);
// and
let casper_key = CasperPrivateKey::from_key(public_key);
// then there could also be
let casper_key = CasperPrivateKey::new(public_key);
```
but the main issue is that this method doesn't do anything.

Reply:
If you compare it with `from_file`: `from_file` provides some funcitonality

Reply:
This was abstraction leftover, cleaned in https://github.com/cspr-rad/kairos/pull/17/commits/0744fd55493fc154bb0f2bbd9d608d922501a5b6.